LICENSE

```diff
@@ -295,6 +295,10 @@ Apache Geode bundles the following files under the MIT license:
   - Timeago v1.3.0 (http://timeago.yarp.com), Copyright (c) 2008-2015 Ryan
     McGeary
   - zTree v3.5.02 (http://zTree.me/), Copyright (c) 2010 Hunter.z
+  - Animal Sniffer v1.17 (http://www.mojohaus.org/animal-sniffer), Copyright
+    (c) 2008–2017 MojoHaus.
+  - Checker Qual v2.7.0 (https://checkerframework.org), Copyright
```

Comment:
This seems the same as Animal Sniffer as it's used only by the geode-web and geode-web-management subprojects.